docs/medical-api/api-reference/document/post-upload-url.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: "Upload Document"
+description: "Returns a URL to use for a medical document upload to our servers."
+api: "POST /medical/v1/document/upload"
+---
+
+This endpoint returns a URL to enable you to upload your patients' medical documents, making them available to other HIEs.
+
+## Overview
+
+To use this endpoint, you will need to follow these steps:
+
+1. Create a `DocumentReference` to describe the document that's being uploaded (see sections below for details on the expected fields and format).
+
+2. Execute the endpoint and receive a URL to use for the document upload.
+
+3. Upload the document by executing a `PUT` request using the `URL` with a `Content-Length` header specifying the size of the document.
+
+## Query Params
+
+<ParamField query="patientId" type="string" required>
+  The ID of the Patient.
+</ParamField>
+
+## Body
+
```

Comment:
I just realized, nowhere do we say that the body payload is actually a `DocumentReference` resource, and we're just describing the recommended fields here.

Let's explicitly mention that, and link to the DocumentReference definition?